src/main/java/uk/gov/hmcts/reform/jps/services/SittingRecordService.java

```diff
@@ -77,32 +86,56 @@ public void saveSittingRecords(String hmctsServiceCode,
             .forEach(recordSittingRecord -> {
                 uk.gov.hmcts.reform.jps.domain.SittingRecord sittingRecord =
                     uk.gov.hmcts.reform.jps.domain.SittingRecord.builder()
-                        .sittingDate(recordSittingRecord.getSittingDate())
-                        .statusId(StatusId.RECORDED.name())
-                        .regionId(recordSittingRecord.getRegionId())
+                        .am(Optional.ofNullable(recordSittingRecord.getDurationBoolean())
+                                .map(DurationBoolean::getAm).orElse(false))
+                        .contractTypeId(recordSittingRecord.getContractTypeId())
                         .epimsId(recordSittingRecord.getEpimsId())
                         .hmctsServiceId(hmctsServiceCode)
                         .personalCode(recordSittingRecord.getPersonalCode())
-                        .contractTypeId(recordSittingRecord.getContractTypeId())
                         .judgeRoleTypeId(recordSittingRecord.getJudgeRoleTypeId())
-                        .am(Optional.ofNullable(recordSittingRecord.getDurationBoolean())
-                                .map(DurationBoolean::getAm).orElse(false))
                         .pm(Optional.ofNullable(recordSittingRecord.getDurationBoolean())
-                                .map(DurationBoolean::getPm).orElse(false))
+                            .map(DurationBoolean::getPm).orElse(false))
+                        .regionId(recordSittingRecord.getRegionId())
+                        .sittingDate(recordSittingRecord.getSittingDate())
+                        .statusId(StatusId.RECORDED.name())
                         .build();
 
                 recordSittingRecord.setCreatedDateTime(LocalDateTime.now());
 
                 StatusHistory statusHistory = StatusHistory.builder()
+                    .changedByName(recordSittingRecordRequest.getRecordedByName())
+                    .changedByUserId(recordSittingRecordRequest.getRecordedByIdamId())
+                    .changedDateTime(recordSittingRecord.getCreatedDateTime())
                     .statusId(StatusId.RECORDED.name())
-                    .changeDateTime(recordSittingRecord.getCreatedDateTime())
-                    .changeByUserId(recordSittingRecordRequest.getRecordedByIdamId())
-                    .changeByName(recordSittingRecordRequest.getRecordedByName())
                     .build();
 
                 sittingRecord.addStatusHistory(statusHistory);
                 sittingRecordRepository.save(sittingRecord);
             });
     }
 
+    private String getAccountCode(String hmctsServiceCode) {
+        if (null == serviceService) {
+            LOGGER.info("serviceService is NULL!");
+            return null;
+        }
+
+        uk.gov.hmcts.reform.jps.domain.Service service = serviceService.findService(hmctsServiceCode);
+        if (null == service) {
```

Comment:
This could be replaced with Objects.isNull

Reply:
done